websites/documentation/src/content/writing/pages.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: Pages
+wideLayout: true
+---
+
+You can find the respective files of all content pages in `src/content`. All new pages must be created as MDX files in this directory. MDX allows you to use JSX in your Markdown content, in addition to standard Markdown syntax.
+
+The page URL corresponds to the file system path by default. For example, an MDX file in `src/content/getting-started/installation.mdx` will render a page in `docs.commercetools.com/getting-started/installation`.
+
+You can freely organize pages according to the website structure in `src/data/navigation.yaml`.
+
+# Markdown frontmatter
+
+All content pages must contain some metadata enclosed within the `---` lines in the frontmatter section. The frontmatter section is a leading section of a file that allows you to add key-value pairs that define the properties of a content page.
+
+```mdx
+---
+title: The title of this page
+---
+
+This is the actual page content.
+```
+
+We support the following key-value pairs in the frontmatter section:
+
+- `title` (string): use (mandatorily) to define the title of a page. It can have the same value as the title in `src/data/navigation.yaml`, or longer.
+- `beta` (boolean): use to indicate if <Beta /> should be displayed for an entire page.
+
+  <Info>
+
+  You can use <Beta /> in other places of a file apart from the frontmatter section. For more information, see [Beta section](/writing/sample#beta-section-).
+
+  </Info>
+
+- `excludeFromSearchIndex` (boolean): use to exclude a page from being indexed by web crawlers. This option effectively sets the `robots="noindex"` meta attribute.
+
+  <Warning>
+
+  This doesn't work with Algolia's Docsearch crawler currently.
+
+  </Warning>
+
+- `navLevels` (number): use to reduce the depth of right-hand index navigation for pages where it could get too long to fit all headings in the screen. You must set to 2. For more information, see [Child section navigation](/writing/navigation#child-section-navigation).
```

Comment:
```suggestion
- `navLevels` (number): use to reduce the depth of right-hand index navigation for pages where it could get too long to fit all headings in the screen. In addition the [Child section navigation](/writing/navigation#child-section-navigation) helps you to keep the index navigation short.
```